src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random randomIndex = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[randomIndex.nextInt(Color.values().length)].name();
```

Comment:
That line looks too long. Look for to a good practise [rule](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html#:~:text=4.1%20Line%20Length,no%20more%20than%2070%20characters.).
Also, define `Color.values()` array as a class field. It would provide better performance.